### DISCUSSION TITLE
Spirit Owl + 3D Inpainting

### DISCUSSION THREAD

**Discussion by gateway** [rkhamilton/vqgan-clip-generator#60] [Share generated art]
So I thought I would share some stuff I'm working on, its a combo of VQGAN +CLIP for some of my spirit animals im creating and then I process them though 3D Photo Inpainting process from https://github.com/vt-vl-lab/3d-photo-inpainting

Here is the first test result.

https://youtu.be/KTsPnnbzuzg

Unfortunately I couldn't get the 3D Photo Inpainting to work on windows but I had it already installed on my ubuntu box ages ago so I thought I would see how well the depth estimation works on some of my creations.

**Comment by rkhamilton** [comment]
Looks great! I had never seen that 3D inpainting stuff before.

**Reply by gateway** [reply]
yea its pretty neat.. creates a nice depth map that you can then use on FB to show your photo as sorta 3d.. Just looking at ways to combined a few things to work though some video ideas I have. I'm still having a hard time creating a semi smooth transition from my original image to iteration x version but ill keep plugging away.
